CMakeLists.txt

```diff
@@ -190,49 +195,76 @@ ELSE()
 	MESSAGE(FATAL_ERROR "Please get them (www.python.org)")
 ENDIF()
 
-INCLUDE(FindSDL)
-IF(SDL_FOUND)
-	MESSAGE(STATUS "Looking for SDL1: found")
-ELSE()
-	MESSAGE(WARNING "Looking for SDL1: not found!")
+IF(SDL_BACKEND STREQUAL "SDL")
+	INCLUDE(FindSDL)
+	IF(USE_SDLMIXER)
+		FIND_PACKAGE(SDL_mixer MODULE)
+	ENDIF()
 ENDIF()
 
-# look for sdl2 if sdl1 was not found or a preference was stated
-IF(USE_SDL2 OR NOT SDL_FOUND)
+# FIXME: Compatibility - remove USE_SDL2 in the future
+IF((SDL_BACKEND STREQUAL "SDL2") OR USE_SDL2)
```

Comment:
Default is still SDL 1.2. It's defined here (line 32):
```SET(SDL_BACKEND "SDL" CACHE STRING "libSDL version (possible values: SDL or SDL2)")```
So, if anyone have both libraries, cmake without options will choose SDL 1.2 as intended earlier. For choosing SDL2 there need to add -DUSE_SDL2 (old behavior) or -DSDL_BACKEND=SDL2 (new one).